README.md

```diff
@@ -36,7 +43,26 @@ And put this in your config to load rust-mode automatically:
 
 `(require 'rust-mode)`
 
-# Manual Installation
+## use-package and straight
+
+```elisp
+(defvar bootstrap-version)
+(let ((bootstrap-file
+       (expand-file-name "~/tmp/git/straight.el/bootstrap.el" user-emacs-directory))
+      (bootstrap-version 5))
+  (unless (file-exists-p bootstrap-file)
+    (with-current-buffer
+        (url-retrieve-synchronously
+         "https://raw.githubusercontent.com/raxod502/straight.el/develop/install.el"
+         'silent 'inhibit-cookies)
+      (goto-char (point-max))
+      (eval-print-last-sexp)))
+  (load bootstrap-file nil 'nomessage))
```

Comment:
this can be found, up to date, in https://github.com/raxod502/straight.el.

I don't think this is always correct (what if "~/tmp/" does not exist?).
Maybe you can just mention that this is an option that exists and redirect to one resource rather than putting that in here.

Reply:
True.